appveyor build fails linking winapi

```
  Compiling winapi v0.3.5
     Running `rustc --crate-name semver C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\semver-0.1.20\src\lib.rs --crate-type lib --emit=dep-info,link -C debuginfo=2 -C metadata=fe0cd2373c8d49f2 -C extra-filename=-fe0cd2373c8d49f2 --out-dir C:\projects\servo\target\debug\deps -L dependency=C:\projects\servo\target\debug\deps --cap-lints allow -Dwarnings -W unused-extern-crates`
     Running `rustc --crate-name build_script_build C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\winapi-0.3.5\build.rs --crate-type bin --emit=dep-info,link -C debuginfo=2 --cfg "feature=\"basetsd\"" --cfg "feature=\"combaseapi\"" --cfg "feature=\"consoleapi\"" --cfg "feature=\"dwmapi\"" --cfg "feature=\"errhandlingapi\"" --cfg "feature=\"fileapi\"" --cfg "feature=\"handleapi\"" --cfg "feature=\"hidusage\"" --cfg "feature=\"knownfolders\"" --cfg "feature=\"libloaderapi\"" --cfg "feature=\"memoryapi\"" --cfg "feature=\"minwindef\"" --cfg "feature=\"ntsecapi\"" --cfg "feature=\"ntstatus\"" --cfg "feature=\"objbase\"" --cfg "feature=\"processenv\"" --cfg "feature=\"processthreadsapi\"" --cfg "feature=\"profileapi\"" --cfg "feature=\"shellapi\"" --cfg "feature=\"shellscalingapi\"" --cfg "feature=\"shlobj\"" --cfg "feature=\"shobjidl_core\"" --cfg "feature=\"std\"" --cfg "feature=\"synchapi\"" --cfg "feature=\"sysinfoapi\"" --cfg "feature=\"timezoneapi\"" --cfg "feature=\"unknwnbase\"" --cfg "feature=\"winbase\"" --cfg "feature=\"wincon\"" --cfg "feature=\"windowsx\"" --cfg "feature=\"winerror\"" --cfg "feature=\"wingdi\"" --cfg "feature=\"winnt\"" --cfg "feature=\"winuser\"" -C metadata=c31d73318e3af517 -C extra-filename=-c31d73318e3af517 --out-dir C:\projects\servo\target\debug\build\winapi-c31d73318e3af517 -L dependency=C:\projects\servo\target\debug\deps --cap-lints allow -Dwarnings -W unused-extern-crates`
   Compiling winapi-build v0.1.1
     Running `rustc --crate-name build C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\winapi-build-0.1.1\src\lib.rs --crate-type lib --emit=dep-info,link -C debuginfo=2 -C metadata=0d75294bd86a2cb8 -C extra-filename=-0d75294bd86a2cb8 --out-dir C:\projects\servo\target\debug\deps -L dependency=C:\projects\servo\target\debug\deps --cap-lints allow -Dwarnings -W unused-extern-crates`
error: linking with `C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\bin\HostX64\x64\link.exe` failed: exit code: 1181
  |
  = note: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\VC\\Tools\\MSVC\\14.14.26428\\bin\\HostX64\\x64\\link.exe" "/NOLOGO" "/NXCOMPAT" "/LIBPATH:C:\\Users\\appveyor\\.rustup\\toolchains\\nightly-2018-07-17-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build0.rcgu.o" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build1.rcgu.o" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build10.rcgu.o" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build11.rcgu.o" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build12.rcgu.o" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build13.rcgu.o" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build14.rcgu.o" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build15.rcgu.o" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build2.rcgu.o" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build3.rcgu.o" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build4.rcgu.o" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build5.rcgu.o" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build6.rcgu.o" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build7.rcgu.o" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build8.rcgu.o" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.build_script_build9.rcgu.o" "/OUT:C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.exe" "C:\\projects\\servo\\target\\debug\\build\\winapi-c31d73318e3af517\\build_script_build-c31d73318e3af517.crate.allocator.rcgu.o" "/OPT:REF,NOICF" "/DEBUG" "/NATVIS:C:\\Users\\appveyor\\.rustup\\toolchains\\nightly-2018-07-17-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\intrinsic.natvis" "/NATVIS:C:\\Users\\appveyor\\.rustup\\toolchains\\nightly-2018-07-17-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\liballoc.natvis" "/NATVIS:C:\\Users\\appveyor\\.rustup\\toolchains\\nightly-2018-07-17-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libcore.natvis" "/LIBPATH:C:\\projects\\servo\\target\\debug\\deps" "/LIBPATH:C:\\Users\\appveyor\\.rustup\\toolchains\\nightly-2018-07-17-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "C:\\Users\\appveyor\\.rustup\\toolchains\\nightly-2018-07-17-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libstd-013ec3941280e756.rlib" "C:\\Users\\appveyor\\.rustup\\toolchains\\nightly-2018-07-17-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libpanic_unwind-e0af97a326ee2088.rlib" "C:\\Users\\appveyor\\.rustup\\toolchains\\nightly-2018-07-17-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libunwind-2e215325d8b57fd5.rlib" "C:\\Users\\appveyor\\.rustup\\toolchains\\nightly-2018-07-17-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\liblibc-2022d6da1cf82a6f.rlib" "C:\\Users\\appveyor\\.rustup\\toolchains\\nightly-2018-07-17-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\liballoc_system-1da4be6dab3578f4.rlib" "C:\\Users\\appveyor\\.rustup\\toolchains\\nightly-2018-07-17-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\liballoc-b7344f562e07e8a0.rlib" "C:\\Users\\appveyor\\.rustup\\toolchains\\nightly-2018-07-17-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcore-d8b630bd33ea051c.rlib" "C:\\Users\\appveyor\\.rustup\\toolchains\\nightly-2018-07-17-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcompiler_builtins-8b5c5275551b24cd.rlib" "advapi32.lib" "ws2_32.lib" "userenv.lib" "shell32.lib" "msvcrt.lib"
  = note: LINK : fatal error LNK1181: cannot open input file 'advapi32.lib'
          
error: aborting due to previous error
error: Could not compile `winapi`.
Caused by:

```

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -54,8 +54,8 @@ install:
   - rustup-init.exe -y --default-host x86_64-pc-windows-msvc --default-toolchain none
   - set PATH=%PATH%;C:\Users\appveyor\.cargo\bin
   - set PKG_CONFIG_PATH=%PKG_CONFIG_PATH%;C:\gstreamer\1.0\x86_64\lib\pkgconfig
-  - set LIB=C:\gstreamer\1.0\x86_64\lib
-  - set LIBPATH=C:\gstreamer\1.0\x86_64\lib
+  - set LIB=%LIB%;C:\gstreamer\1.0\x86_64\lib
+  - set LIBPATH=C:\gstreamer\1.0\x86_64\lib;%LIBPATH%
   - rustup -V
   - mach rustc --version
   - mach cargo --version

